src/fabric/api/fixed-asset-base/go/contracts/FixedAssetContract.go

```diff
@@ -125,6 +125,38 @@ func (contract *FixedAssetContract) GetAssetsFromBatch(ctx utils.Context, batch
 	return &retArr, nil
 }
 
+//Delete an Asset from the registry that was created by createAsset
+func (s *FixedAssetContract) DeleteAsset(ctx utils.Context, uuid string) error {
+	fmt.Printf("Entering deleteAsset")
+	fmt.Printf(`Returning result for deleteAsset with uuid: ${uuid}`)
+
+	err := ctx.GetStub().DelState(uuid)
+
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+//Delete batch of assets from the registry that was created by createAsset
+func (s *FixedAssetContract) DeleteAssetsFromBatch(ctx utils.Context, batch []string) error {
+	fmt.Printf("Entering deleteAssetsFromBatch()")
+
+	for _, uuid := range batch {
+		fmt.Printf(`deleting UUID ${uuid}`)
+		err := ctx.GetStub().DelState(uuid)
+
+		if err != nil {
+			return err
+		}
+	}
+
+	fmt.Printf(`Exiting deleteAssetsFromBatch()`)
```

Comment:
Use PrintLn. Could you check the rest of the implementation to make sure all the methods output a newline